tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -277,26 +278,32 @@ def fit(train_data,
                             min_time_limit: (float, default=0) Allow this model to train for at least this long (in sec), regardless of the time limit it would otherwise be granted.
                                 If `min_time_limit >= max_time_limit`, time_limit will be set to min_time_limit.
                                 If `min_time_limit=None`, time_limit will be set to None and the model will have no training time restriction.
+                    AG_args_ensemble: Dictionary of ensemble customization options. Only models in stack levels >=1 are impacted by these parameters.
```

Comment:
"ensemble customization options" could be more clearly written as something like:
"hyperparameters shared by all models that control how they are ensembled".

Reply:
added